### DISCUSSION TITLE
Which musical instrument's name translates to "wood sounds"? 🪵🔊

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1235] [Galaxy Trivia] [answered]
**Bonus:** What is the world's only gem made of one element?

**Comment by peterbaccaro** [comment, marked as answer]
The musical instrument the translates to "wood sounds" is the **xylophone**.

**Diamond** is the only gem made of a single element: It is typically about 99.95 percent carbon. The other 0.05 percent can include one or more trace elements, which are atoms that aren't part of the diamond's essential chemistry.
